package.json

```diff
@@ -20,23 +20,32 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
-    "eslint": "^4.19.1",
+    "eslint": "^5.1.0",
     "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-mocha": "^5.0.0",
-    "eslint-plugin-prettier": "^2.6.0",
+    "eslint-plugin-mocha": "^5.1.0",
+    "eslint-plugin-prettier": "^2.6.2",
     "mailparser": "^2.2.0",
     "mocha": "^5.2.0",
-    "prettier": "^1.13.5",
+    "prettier": "^1.13.7",
+    "rollup": "^0.62.0",
+    "rollup-plugin-commonjs": "^9.1.3",
+    "rollup-plugin-node-resolve": "^3.3.0",
     "smtp-server": "^3.4.6"
   },
   "engine": [
     "node >= 6"
   ],
   "main": "email.js",
```

Comment:
if we move this library over to use the build, does it just require changing "main" ? could you walk me through what we would need to do and what the impact of switching to this is for end users?

Reply:
yes, `main` would need to be updated to point at the rollup build. end users shouldn't notice a difference* or need to adjust their own code.

*one relatively minor exception: the bundling apparently puts `console` into the global scope for the `text-encoding` library, which leads to a bunch of unnecessary logging about some piece of upstream code specifying an encoding in the constructor. harmlessly annoying and totally fixable on its own (probably by doing AST parsing/string replacement with the output from rollup's JS API) but wasn't a priority to me for this PR.